bin/setup

```diff
@@ -1,32 +1,19 @@
-#!/usr/bin/env ruby
 require 'pathname'
 require 'fileutils'
 include FileUtils
 
-# path to your application root.
 APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
 
 def system!(*args)
   system(*args) || abort("\n== Command #{args} failed ==")
 end
 
 chdir APP_ROOT do
-  # This script is a starting point to setup your application.
-  # Add necessary setup steps to this file.
 
   puts '== Installing dependencies =='
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
-  # Install JavaScript dependencies if using Yarn
-  # system('bin/yarn')
-
-
-  # puts "\n== Copying sample files =="
-  # unless File.exist?('config/database.yml')
-  #   cp 'config/database.yml.sample', 'config/database.yml'
-  # end
-
```

Comment:
I would also keep some of these comments, like line 14-15.
But then again YMMV, whatever you prefer

Reply:
Also double quotes...